blueprints/ember-frost-tabs/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
 
   afterInstall: function (options) {
     const addonsToAdd = [
-      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-frost-core', target: '1.23.9'},
```

Comment:
Because we want to ensure the version of Ember CLI things are being built against, with this version pinning to `2.12.3`.  We will also be re-evaluating our approach of using blueprints to manage dependencies so either format of these entries are likely to be removed in the near future.